extension/content_script.js

```diff
@@ -470,6 +470,9 @@ function toggleKeywordSettings() {
 		settingsView.changeSetting('keyword', 'bright'); 
 	}
 	else if (currentKeywordStyle == 'bright') { 
+		settingsView.changeSetting('keyword', 'gentle'); 
```

Comment:
dont have to do this but fyi a nice one-liner

keywordStyleOrders = [off, light, bright, gentle]
nextStyle = keywordStyleOrders[(keywordStyleOrders.indexOf(currentStyle) + 1) % keywordStyleOrders.length]

Reply:
V cool idea, thanks!